src/Apps/Auction/Routes/__fixtures__/MutationResults/persistBidderRequirements.ts

```diff
@@ -0,0 +1,35 @@
+import { RegisterPreCreateBidderMutationResponse } from "__generated__/RegisterPreCreateBidderMutation.graphql"
+
+export const persistBidderRequirementsSuccessful: RegisterPreCreateBidderMutationResponse = {
+  updateMyUserProfile: {
+    user: {
+      id: "example-user-id",
+    },
+  },
+  createCreditCard: {
+    creditCardOrError: {
+      creditCardEdge: {
+        node: {
+          last_digits: "4242",
+        },
+      },
+    },
+  },
+}
+
+export const persistBidderRequirementsFailed: RegisterPreCreateBidderMutationResponse = {
+  updateMyUserProfile: {
```

Comment:
is it possible/sensible for the `updateMyUserProfile` to return an error? It doesn't seem like it's modeled in the MP API, so I presume not, meaning that, I _believe_ [this `catch`](https://github.com/artsy/reaction/blob/53e90c8ee71ba37177c2d30b49ef2d908e3dd55a/src/Apps/Auction/Routes/Register/index.tsx#L181) would rescue an exception.

Reply:
> is it possible/sensible for the updateMyUserProfile to return an error?

Possible, but that seems out of scope for this PR. At the moment, the mutation doesn't model an error. Any error thrown by metaphysics should be caught by the top-level `errors` handling or the catch you linked to.